core/src/prediction/OptimizedPredictionStrategy.h

```diff
@@ -86,7 +86,16 @@ class OptimizedPredictionStrategy {
       const std::vector<std::vector<size_t>>& leaf_samples,
       const Observations& observations) = 0;
 
-  virtual std::vector<double> compute_debiased_error(
+ /**
+  * Computes out-of-bag error for a single sample.
```

Comment:
It would be good to describe what each item of the pair represents, since this is really not obvious (the debiased error, followed by the estimated bias).